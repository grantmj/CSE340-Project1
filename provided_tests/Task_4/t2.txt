TASKS 
    1 4

POLY 
    F(a) = a^2 + (a^3 + 7a^3);

EXECUTE 
    INPUT x;

INPUTS 
    1