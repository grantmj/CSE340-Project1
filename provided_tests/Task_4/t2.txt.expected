POLY - COMBINED MONOMIAL LISTS
	F(a) = a^2 + (8 a^3);
